packages/eslint-config/package.json

```diff
@@ -62,5 +62,5 @@
   "scripts": {
     "lint": "eslint ."
   },
-  "version": "0.0.2"
+  "version": "0.0.3"
```

Comment:
I'd argue that you might be included to cut a `0.1.0` just to make it easier to get these upgrades without having to manually update the module